contracts/src/v0.6/Oracle.sol

```diff
@@ -97,6 +107,13 @@ contract Oracle is ChainlinkRequestInterface, OracleInterface, Ownable, LinkToke
       )
     );
 
+    s_packedRequests[requestId] = PackedOracleRequest(
+      _callbackAddress,
+        _callbackFunctionId,
+      uint128(_payment),
+      uint128(expiration)
+    );
```

Comment:
Summary: Writing to storage on request. Still emitting values in event, so no breaking changes.